core/display.js

```diff
@@ -746,12 +746,18 @@
             this._renderQ.push(action);
             if (this._renderQ.length === 1) {
                 // If this can be rendered immediately it will be, otherwise
-                // the scanner will start polling the queue (every
-                // requestAnimationFrame interval)
+                // the scanner will wait for the relevant event
                 this._scan_renderQ();
             }
         },
 
+        _resume_renderQ: function() {
+            // "this" is the object that is ready, not the
```

Comment:
is there a significant performance penalty to using bind here instead (there might have been at some point -- it's worth investigating)?  Using bind would probably make this code a bit clearer (not that the comment is bad -- it just looks weird).